### PR TITLE
Update Hugo to 76.5 and theme to 111

### PR DESCRIPTION
**Pull request by flora8984461**
Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ pluralizeListTitles = false
   keywords = ["eclipse", "research"]
   styles = "css/styles.css"
   gcse = "011805775785170369411:p3ec0igo0qq"
+  # hugo_js = "js/solstice.js"
```
Comment (autumnfound) [inline config.toml]: why add this line?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 matrix:
   include:
-    - node_js: 10
+    - node_js: 12
```
Comment (autumnfound) [inline .travis.yml]: probably removing this section, but holding for confirmation from Chris

File: config.toml (hunk 1 above)
Comment (flora8984461) [reply config.toml]: Oops, btw, for this site it's also showing the err of "Not found", I'm thinking of adding a new empty one hugo_js = ""

Comment (flora8984461) [reply config.toml]: Ah, I think it only appears on localhost, I don't need it, Sorry

**Review by autumnfound — APPROVED** [review]
+1 as long as Travis builds successfully

**Pull request merged**